src/pages/contact-me.jsx

```diff
@@ -1,7 +1,11 @@
 import React from "react";
+import styled from "styled-components";
 
 function ContactMe() {
-    return <div>Get in touch!</div>;
+    const ContactMe = styled.div `
```

Comment:
Same change mentioned in [this comment](https://github.com/rommelpatel/website/pull/4/commits/27fc77fbde943dbd6785f0ce58938c9aa9b525aa#r1027606557)